master/internal/db/postgres_trial_logs.go

```diff
@@ -36,14 +36,21 @@ SELECT
     END AS message,
     l.agent_id,
     l.container_id,
-    l.timestamp,
+    CASE
```

Comment:
non-blocking: This could use `coalesce`, which [I believe](https://stackoverflow.com/questions/505971/coalesce-guaranteed-to-short-circuit) should evaluate the same way (i.e., not evaluate the second argument any more eagerly).